testing/functional_tests_assertions.rst

```diff
@@ -19,17 +19,17 @@ Now here is the example with the assertions specific to Symfony::
 
     $this->assertSelectorTextContains('html', 'Hello World');
 
-.. note::
-
-    These assertions only work if a request has been made with the ``Client``
-    in a test case extending the ``WebTestCase`` class.
-
 Assertions Reference
 ---------------------
 
 Response
 ~~~~~~~~
 
+.. note::
+
+    The following assertions only work if a request has been made with the
+    ``Client`` in a test case extending the ``WebTestCase`` class.
```

Comment:
Don't we use `:class:` pointer anymore?

Reply:
Not everywhere because we cannot link to the method anymore on GitHub. It just links to the class. If this will be possible in the future we could enforce this again.

If you like you can use it 👍🏻